src/main/java/com/google/starfish/models/Label.java

```diff
@@ -0,0 +1,21 @@
+package com.google.starfish.models;
+
+/** Enum declaration for type of label. */
+enum Type {
+  SCHOOL,
+  COURSE,
+  OTHER
+}
+
+/** Represents a label */
+public class Label {
+
+  private String title;
+  private Type type;
+
+  public Label(String title, Type type) {
+    if (title == null || type == null) { throw new ArgumentNullException(); }
+    this.title = title;
+    this.type = type;
+  }
+}
```

Comment:
Note necessary for MVP, but we might want to include a `String code` or `String abbreviation` field for the shorthand names of schools and courses. Maybe SchoolLabels inherit from Label? Also sounds like the DB might need to change a bit. Not sure how to go about this, but definitely not for MVP.

Reply:
This is a good point, we could potentially make that change in the future by making the change to the model and schema but I think you're right that it's not too important for now. I think having SchoolLabels and CourseLabels inheriting from Labels is a good potential solution to add those more specific attributes.